### PR TITLE
bpo-44405: make dis cli "official" 

### PR DESCRIPTION
**Pull request by CCLDArjun**
https://bugs.python.org/issue44405

### CODE REVIEW

File: Lib/dis.py [hunk 1]
```diff
@@ -566,7 +566,7 @@ def dis(self):
             return output.getvalue()
 
 
-def _test():
+def main():
```
Comment (eamanu) [inline Lib/dis.py:569]: I wouldn't change the name of the function.

File: Doc/library/dis.rst [hunk 2]
```diff
@@ -41,6 +41,10 @@ the following command can be used to display the disassembly of
 
 (The "2" is a line number).
 
+:mod:`dis` an also be invoked as a script to disassemble other scripts. For example::
```
Comment (eamanu) [inline Doc/library/dis.rst:44]: take care the page width  (79 chars, if I'm not wrong)

Comment (eamanu) [reply Doc/library/dis.rst:44]: Can you add more detail about the args (infile)?

**Review by eamanu — CHANGES_REQUESTED** [review]

File: Lib/dis.py (hunk 1 above)
Comment (eamanu) [reply Lib/dis.py:569]: Sorry I'm just read the comments on bpo about change this name :)

File: Doc/library/dis.rst (hunk 2 above)
Comment (CCLDArjun) [reply Doc/library/dis.rst:44]: I mean, it seems pretty straightforward. What more to explain other than feed it a python script?

**Comment by hugovk** [comment]
New issue for documenting the CLI: https://github.com/python/cpython/issues/108826

And PR: https://github.com/python/cpython/pull/108827

**Pull request closed without merging**